packages/frontend/src/lib/disk-image/DiskImageActions.svelte

```diff
@@ -33,7 +33,11 @@ onMount(async () => {
 <!-- Only show the Terminal button if object.arch actually exists or else we will not be able to pass in the architecture information to the build correctly.
 Only show if on macOS as well as that is the only option we support at the moment -->
 {#if object.arch && !isWindows}
-  <ListItemButtonIcon title="Launch VM" onClick={() => gotoVM()} detailed={detailed} icon={faTerminal} />
+  <ListItemButtonIcon title="Launch VM" onClick={(): Promise<void> => gotoVM()} detailed={detailed} icon={faTerminal} />
```

Comment:
curious here why it's not using `{gotoVM}`

Reply:
🤦🏼‍♂️ whatever I had the first time failed svelte:check so I changed it to this. Fixed in new commit.